src/sdk/namespace_cache.js

```diff
@@ -449,7 +486,14 @@ class NamespaceCache {
                     upload_params.end = hub_read_range.end;
                     // Set object ID since partial object has been created before
                     upload_params.obj_id = params.object_md.obj_id;
+                    upload_params.async_update_cache_stats = async () => {
+                        this.stats_collector.update_cache_stats({
+                            bucket_name: params.bucket,
+                            write_bytes: upload_params.end - upload_params.start,
+                        });
+                    };
```

Comment:
```suggestion
                            update_cache_stats: data_length => this.stats_collector.update_cache_stats({
                                  bucket_name: params.bucket,
                                  write_bytes: data_length,
                             });
```

Reply:
You mean `upload_params.update_cache_stats: data_length => this.stats_collector.update_cache_stats({` ?

Reply:
right, but with an assignment `=`